Warmup-1/loneTeen/test/index.test.ts

```diff
@@ -0,0 +1,23 @@
+// We'll say that a number is "teen" if it is in the range 13..19 inclusive. Given 2 int values, return true if one or the other is teen, but not both.
+
+import { assert } from "chai";
+import { loneTeen } from "../src";
+
+// loneTeen(13, 99) → true
+// loneTeen(21, 19) → true
+// loneTeen(13, 13) → false
+
+describe("loneTeen", () => {
```

Comment:
Please add a test when they're both not teens

Reply:
Sure.